src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/GnosisControlSafeForm.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { FormikProps } from 'formik';
+
+import Avatar from '~core/Avatar';
+import { DialogSection } from '~core/Dialog';
+import Heading from '~core/Heading';
+import ExternalLink from '~core/ExternalLink';
+import Button, { AddItemButton } from '~core/Button';
+import { SingleSafePicker, filterUserSelection } from '~core/SingleUserPicker';
+
+import { GNOSIS_SAFE_INTEGRATION_LEARN_MORE } from '~externalUrls';
+import { Colony } from '~data/index';
+import { Address } from '~types/index';
+
+import { FormValues, transactionOptions } from './GnosisControlSafeDialog';
+import styles from './GnosisControlSafeForm.css';
+import { Select } from '~core/Fields';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.title',
+    defaultMessage: 'Control Safe',
+  },
+  description: {
+    id: 'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.description',
+    defaultMessage: `You can use Control Safe to interact with other third party smart contracts. Be careful. <a>Learn more</a>`,
+  },
+  selectSafe: {
+    id: 'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.selectSafe',
+    defaultMessage: 'Select Safe',
+  },
+  safePickerPlaceholder: {
+    id: `dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.safePickerPlaceholder`,
+    defaultMessage: 'Select safe to control',
+  },
+  transactionLabel: {
+    id: `dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.transactionLabel`,
+    defaultMessage: 'Select transaction type',
+  },
+  transactionPlaceholder: {
+    id: `dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.transactionPlaceholder`,
+    defaultMessage: 'Select transaction',
+  },
+  buttonTransaction: {
+    id: `dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.buttonTransaction`,
+    defaultMessage: 'Add another transaction',
+  },
+  buttonInteract: {
+    id:
+      'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.buttonInteract',
+    defaultMessage: 'Interact',
+  },
+});
+
+const displayName = 'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm';
+
+export interface GnosisSafe {
+  name: string;
+  address: Address;
+  chain: string;
+}
+
+interface Props {
+  colony: Colony;
+  safes: GnosisSafe[];
+  back?: () => void;
+}
+
+const renderAvatar = (address: string, item) => (
+  <Avatar
+    seed={address.toLocaleLowerCase()}
+    size="xs"
+    notSet={false}
+    title={item.name}
+    placeholderIcon="at-sign-circle"
+  />
+);
+
+const GnosisControlSafeForm = ({
+  back,
+  handleSubmit,
+  safes,
+  isSubmitting,
+  isValid,
+}: Props & FormikProps<FormValues>) => {
+  return (
+    <>
+      <DialogSection>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none', theme: 'dark' }}
+          text={MSG.title}
+        />
+      </DialogSection>
```

Comment:
![Screenshot from 2022-06-23 22-07-32](https://user-images.githubusercontent.com/1193222/175378984-289a23eb-f242-4b02-a197-7b1ffb935d2b.png)

I think there should be more space between the heading and the copy, in order to better align with the other modals